lib/client.js

```diff
@@ -2,6 +2,8 @@ var superagent = require("superagent");
 var errors = require("./errors");
 var request = require("request");
 
+var SDK_VERSION_NUMBER = "1.3.1";
```

Comment:
Pull this from the package.json .

We don't want to have to update this value in two places... too easy to get out of sync. 

Check here how we do this in node v2.

[How to pull from package.json](https://github.com/bandwidthcom/node-bandwidth/blob/2.0-preview/lib/client.js#L4)